devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -76,9 +76,9 @@ def parseAttrMessage(description) {
 		map = parseAduroSmartButtonMessage(descMap)
     	} else if (descMap?.clusterInt == zigbee.ONOFF_CLUSTER && descMap.isClusterSpecific) {
 		map = getButtonEvent(descMap)
-	} else if(descMap?.clusterInt == 0xFC80) {
+	} else if(descMap?.clusterInt == 0x0005) {
```

Comment:
How does this affect the other buttons in this DTH?

Reply:
the if branch for cluster int 0xFC80 was added by meifen months ago, specifically for this heiman multi-button device, after I changed configure function for the device, the clusterInt changed to 0x0005, so this statement only works for heiman multi-button, won't affect other devices

Reply:
I guess the last thing I would check is to make sure that these two devices don't suddenly mis-behave:
```
fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", outClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", manufacturer: "AduroSmart Eria", model: "ADUROLIGHT_CSC", deviceJoinName: "Eria scene button switch V2.1", mnmn: "SmartThings", vid: "generic-4-button"
fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", outClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", manufacturer: "ADUROLIGHT", model: "ADUROLIGHT_CSC", deviceJoinName: "Eria scene button switch V2.0", mnmn: "SmartThings", vid: "generic-4-button"
```

Also, nit: spacing between the `if` and `(`: `} else if(`

Reply:
add space in new commit

and from the logic, the two devices will be handled in previous case:
else if (isAduroSmartRemote()) {//ADUROLIGHT_CSC is defined in isAduroSmartRemote
		map = parseAduroSmartButtonMessage(descMap)
    	}